book/src/core-testing-doc/api/db-sync-explorer/postman_collection.json

```diff
@@ -0,0 +1,158 @@
+{
+	"info": {
+		"_postman_id": "cb8ec9f2-ba0d-4840-a1ce-b828936795f1",
+		"name": "Db Sync Explorer",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "11181958"
+	},
```

Comment:
Is _exporter_id and _postman_id starting with underscore something wanted?

Reply:
this is autogenerated file from postman, i think it's better not to touch it